.github/CODEOWNERS

```diff
@@ -33,3 +33,6 @@
 
 /packages/itwin/imodel-react-hooks  @itwin/viewer-components-reviewers @michaelbelousov
 /common/changes/@itwin/imodel-react-hooks  @itwin/viewer-components-reviewers @michaelbelousov
+
+/packages/itwin/infrastructure-iot  @itwin/viewer-components-reviewers @AntonAI
```

Comment:
is there anyone else on the team that can help review or maintain the code?

Reply:
Added more team members to list of reviewers